core/src/consensus.rs

```diff
@@ -3161,8 +3207,8 @@ pub mod test {
     #[test]
     fn test_adjust_lockouts_after_replay_time_warped() {
         let mut tower = Tower::new_for_tests(10, 0.9);
-        tower.vote_state.votes.push_back(Lockout::new(1));
-        tower.vote_state.votes.push_back(Lockout::new(0));
+        tower.vote_state.votes.push_back(Lockout::new(1).into());
```

Comment:
generally prefer `IntoType::from(from_value)` over `from_value.into()` for the improved self-documentation

Reply:
That is a good idea.  I'll make the change.